src/components/JobBuild.jsx

```diff
@@ -18,12 +18,27 @@ class JobBuild extends Component {
             </div>
         );
     }
+
+    buildResult() {
```

Comment:
That makes sense. I think breaking this out into its own component will also make it easier to reuse this functionality for other Jenkins widgets (assuming that the histogram and job status widgets encounter the same data from the Jenkins API).

Maybe it would even make sense down the road to extract models or decorators around Jenkins' job or build API objects that encapsulate this logic, allow for easy testing, and allow for reuse throughout the codebase. I don't have too much familiarity with React or JavaScript patterns in this sense, but something like:

```es6
class BuildDecorator {

    const JENKINS_BUILD_STATUS_BUILDING = 'BUILDING';
    const JENKINS_BUILD_STATUS_UNKNOWN  = 'UNKNOWN';

    constructor(build) {
        this.build = build;
    }

    getBuildStatus() {
        if (this.build.result) {
            return this.build.result;
        }

        return this.build.building ? JENKINS_BUILD_STATUS_BUILDING : JENKINS_BUILD_STATUS_UNKNOWN;
    };
}
```